app/models/authenticator/authenticator.rb

```diff
@@ -15,6 +15,10 @@ def initialize(account:, service_id:, required_request_parameters: nil)
       end
     end
 
+    def authenticator_name
+      return self.service_id
```

Comment:
In Ruby the last line is returned, so the `return` is redundant here.  

Alternatively, we can just add an alias:
```ruby
module Authenticator
  class Authenticator
    alias authenticator_name, service_id
    ...
  end
end
```